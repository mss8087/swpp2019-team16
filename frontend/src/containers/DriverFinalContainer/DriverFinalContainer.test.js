import React from 'react';
import DriverFinalContainer from './DriverFinalContainer';
import { renderWithRedux } from '../../test/utils';
import { MemoryRouter, Route } from 'react-router-dom';

describe('<DriverFinalContainer />', () => {
  it('SHOULD match with snapshot', async () => {
    const { container } = renderWithRedux(
      <MemoryRouter initialEntries={['/drivefinal']}>
        <Route component={DriverFinalContainer} path="/driverfinal" />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
