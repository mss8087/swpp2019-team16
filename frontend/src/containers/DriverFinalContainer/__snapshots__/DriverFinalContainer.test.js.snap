// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DriverFinalContainer /> SHOULD match with snapshot 1`] = `<div />`;
